worker/backup_ee.go

```diff
@@ -171,12 +171,16 @@ func ProcessBackupRequest(ctx context.Context, req *pb.BackupRequest, forceFull
 	defer cancel()
 
 	errCh := make(chan error, len(state.Groups))
+	var dropOperations []*pb.DropOperation
 	for _, gid := range groups {
 		br := proto.Clone(req).(*pb.BackupRequest)
 		br.GroupId = gid
 		br.Predicates = predMap[gid]
 		go func(req *pb.BackupRequest) {
-			_, err := BackupGroup(ctx, req)
+			res, err := BackupGroup(ctx, req)
+			if res != nil && len(res.DropOperations) > 0 {
```

Comment:
You don't need to check the length. See https://play.golang.org/p/odlQYbwcQL2

Reply:
Done